.github/workflows/rpm.yml

```diff
@@ -51,6 +51,10 @@ jobs:
           ref: ${{ github.ref }}
           fetch-depth: 0
 
+      - name: Generate build info
+        run: |
+          /bin/python3 scripts/build-info.py data/build-info.json
```

Comment:
Could add this command to the root level Makefile under target named `build-info`.

Then in CI and our dev envs we can say `make build-info`